.github/workflows/ci_any_pr.yaml

```diff
@@ -20,7 +20,7 @@ on:
       - master
 jobs:
   static-code-tests:
-    runs-on: [self-hosted]
+    runs-on: [self-hosted, runner-july-21]
```

Comment:
Is `runner-july-21` a temporary name or the permanent name for this runner?

Reply:
No. This is a label I have added to both the current runners. As we migrate to new runners in future I intend to label them `runner-month-year`